lib/livebook_web/live/hub/edit/fly_component.ex

```diff
@@ -8,20 +8,31 @@ defmodule LivebookWeb.Hub.Edit.FlyComponent do
   def update(assigns, socket) do
     changeset = Fly.change_hub(assigns.hub)
     {:ok, app} = FlyClient.fetch_app(assigns.hub)
+    env_vars = env_vars_from_secrets(app["secrets"])
+
+    {env_var, env_op} =
+      if key = assigns.params["env_var_id"],
```

Comment:
Passing params to a component feels a little brittle to me, instead we could require `env_var_id` assign and let the parent LV read the params itself. wdyt?

Reply:
I've been trying to not use the LV to get specific-component data, but I think it's not a big deal to have the `env_var_id` from LV and pass it directly to the component

Reply:
Yeah, but the other way round it's the component getting LV-specific data pretty much :D